conf/messages.en

```diff
@@ -88,66 +89,83 @@ unauthorised.paragraph.1.link = register for Plastic Packaging Tax
 unauthorised.paragraph.2 = Visit the {0} for more information.
 unauthorised.paragraph.2.link = Plastic Packaging Tax guidance
 
-amendDirectExportPlasticPackaging.title = How much of your finished plastic packaging components did you directly export? - Submit return - Plastic Packaging Tax - GOV.UK
-amendDirectExportPlasticPackaging.heading = How much of your finished plastic packaging components did you directly export?
-amendDirectExportPlasticPackaging.checkYourAnswersLabel = Weight of directly exported plastic packaging
-amendDirectExportPlasticPackaging.error.nonNumeric = Amount must be entered as numbers only
+amendDirectExportPlasticPackaging.title = How much of your finished plastic packaging components did you export, or do you intend to export within 12 months? - Submit return - Plastic Packaging Tax - GOV.UK
+amendDirectExportPlasticPackaging.heading = How much of your finished plastic packaging components did you export, or do you intend to export within 12 months?
+amendDirectExportPlasticPackaging.checkYourAnswersLabel = Plastic packaging exported by you
+amendDirectExportPlasticPackaging.error.nonNumeric = Weight must be entered as numbers only
 amendDirectExportPlasticPackaging.error.required = Enter the weight, in kilograms
-amendDirectExportPlasticPackaging.error.wholeNumber = Enter the weight using whole numbers
+amendDirectExportPlasticPackaging.error.wholeNumber = Weight must not include decimals
 amendDirectExportPlasticPackaging.error.outOfRange.low = Weight must be 0kg or more
 amendDirectExportPlasticPackaging.error.outOfRange.high = Weight must be between 0kg and 99,999,999,999kg
 amendDirectExportPlasticPackaging.change.hidden = AmendDirectExportPlasticPackaging
-amendDirectExportPlasticPackaging.hint = Tax is not charged on plastic packaging that you directly. This will be deducted from your tax liability.
+amendDirectExportPlasticPackaging.para = Plastic packaging you export will show as a deduction on your tax calculation.
+amendDirectExportPlasticPackaging.hint = Enter the weight, in kilograms. 1 tonne is 1,000kg.
 
-amendHumanMedicinePlasticPackaging.title = How much of your finished plastic packaging components were used for the immediate packaging of licensed human medicine? - Submit return - Plastic Packaging Tax - GOV.UK
-amendHumanMedicinePlasticPackaging.heading = How much of your finished plastic packaging components were used for the immediate packaging of licensed human medicine?
-amendHumanMedicinePlasticPackaging.checkYourAnswersLabel = Weight of plastic packaging exported used for licenced human medicines
-amendHumanMedicinePlasticPackaging.error.nonNumeric = Amount must be entered as numbers only
+amendHumanMedicinePlasticPackaging.title = Out of the finished plastic packaging components that you did not export, how much was used for the immediate packaging of licenced human medicines? - Submit return - Plastic Packaging Tax - GOV.UK
```

Comment:
these should use the title wrapper that auto puts these on, 
Also are these submit return or amed return?